client/verta/verta/_internal_utils/_utils.py

```diff
@@ -327,7 +316,7 @@ def get_workspace_name_from_id(self, workspace_id):
         )
 
         workspace = self.must_proto_response(response, Workspace_pb2.Workspace)
-        return workspace.username or workspace.org_name
+        return "{}:{}".format(workspace.org_id, workspace.org_name)
```

Comment:
See my other comment about not needing `org_id`: https://github.com/VertaAI/modeldb/pull/3407/files#r1067587726